### PR TITLE
Add instructions for cabal

### PR DESCRIPTION
**Pull request by hvr**

### CODE REVIEW

File: bin/run-in-xephyr.sh [hunk 1]
```diff
@@ -5,7 +5,17 @@ ARCH=`uname -m`
 OS=`uname -s | tr '[A-Z]' '[a-z]'`
 
 ################################################################################
+if [ -d .stack-work ]; then
+echo "stack build detected"
 BIN_PATH=`stack path --dist-dir`/build/xmonad-testing
+echo "cabal build detected"
```
Comment (pjones) [inline bin/run-in-xephyr.sh]: I think this line needs to be moved down into the `elif`.

File: bin/run-in-xephyr.sh [hunk 2]
```diff
@@ -5,7 +5,17 @@ ARCH=`uname -m`
...
+elif [ -d dist-newstyle ]; then
+BIN_PATH=$(find dist-newstyle/ -type f -executable -name xmonad-testing -printf '%h')
```
Comment (pjones) [inline bin/run-in-xephyr.sh]: Please use backticks instead of `$()` so this script doesn't need bash.

**Review by pjones — CHANGES_REQUESTED** [review]
Great PR, thanks!  I'm looking forward to going back to `cabal`.

Please make the suggested changes and I'll merge.

File: bin/run-in-xephyr.sh (hunk 2 above)
Comment (hvr) [reply bin/run-in-xephyr.sh]: TIL :-)

File: bin/run-in-xephyr.sh (hunk 1 above)
Comment (hvr) [reply bin/run-in-xephyr.sh]: fixed...

**Comment by pjones** [comment]
Thanks again!

**Pull request merged**